scheduler/resource/peer.go

```diff
@@ -389,29 +394,35 @@ func (p *Peer) DeleteStream() {
 }
 
 // Download tiny file from peer
-func (p *Peer) DownloadTinyFile(ctx context.Context) ([]byte, error) {
-	// Download url: http://${host}:${port}/download/${taskIndex}/${taskID}?peerId=scheduler;
+func (p *Peer) DownloadTinyFile() ([]byte, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), downloadTinyFileContextTimeout)
+	defer cancel()
+
+	// Download url: http://${host}:${port}/download/${taskIndex}/${taskID}?peerId=scheduler
 	url := url.URL{
 		Scheme:   "http",
 		Host:     fmt.Sprintf("%s:%d", p.Host.IP, p.Host.DownloadPort),
 		Path:     fmt.Sprintf("download/%s/%s", p.Task.ID[:3], p.Task.ID),
 		RawQuery: "peerId=scheduler",
 	}
-	p.Log.Infof("download tiny file url: %#v", url)
 
 	req, err := http.NewRequestWithContext(ctx, http.MethodGet, url.String(), nil)
 	if err != nil {
 		return []byte{}, err
 	}
 	req.Header.Set(headers.Range, fmt.Sprintf("bytes=%d-%d", 0, p.Task.ContentLength.Load()))
```

Comment:
why req need set range, if req has range header, The data downloaded by CDN has been processed by Range